packages/nextjs/pages/batches/index.tsx

```diff
@@ -172,28 +267,7 @@ const Batches = ({ batchData, openBatchNumber, openBatchStartDate }: PageProps)
       <div className="bg-[#EDEFFF] pt-16 lg:pt-24 pb-16">
```

Comment:
Maybe with `pt-2 lg:pt-8` is enough spacing?

```suggestion
      <div className="bg-[#EDEFFF] pt-2 lg:pt-8 pb-16">
```

Reply:
Changed